src/ui/MAVLinkDecoder.h

```diff
@@ -25,7 +25,7 @@ public slots:
     /** @brief Shift a timestamp in Unix time if necessary */
     quint64 getUnixTimeFromMs(int systemID, quint64 time);
 
-    static const size_t cMessageIds = 256;
+    static const size_t cMessageIds = 512;
```

Comment:
well it actually allocates a buffer that big, so we might need to figure out another way to do this then, allocating an array of 10,000+ pointers might be ugly, not out of the realm of possibility. It would probably be better to use a std::map so it can be dynamically sized.